client/components/plans/plan-price/index.jsx

```diff
@@ -55,7 +55,9 @@ const PlanPrice = React.createClass( {
 			return <div className="plan-price is-placeholder" />;
 		}
 
-		if ( abtest( 'planPricing' ) === 'monthly' && plan.raw_price !== 0 ) {
+		if ( ! plan ) {
+			periodLabel = this.translate( 'Loading' );
```

Comment:
This needs to be a string, but I'll pass through an empty one if this bothers you.